kraken/time_actions/common_time_functions.py

```diff
@@ -79,23 +94,45 @@ def skew_time(scenario):
         if len(pod_names) == 0:
             logging.info("Cannot find pods matching the namespace/label_selector, please check")
             sys.exit(1)
+        pod_counter = 0
         for pod in pod_names:
             if len(pod) > 1:
-                pod_exec(pod[0], skew_command, pod[1])
+                selected_container_name = get_container_name(pod[0], pod[1], container_name)
+                pod_exec_response = pod_exec(pod[0], skew_command, pod[1], selected_container_name)
+                if pod_exec_response is False:
+                    logging.error(
+                        "Couldn't reset time on container %s in pod %s in namespace %s"
+                        % (selected_container_name, pod[0], pod[1])
+                    )
+                    sys.exit(1)
+                pod_names[pod_counter].append(selected_container_name)
             else:
-                pod_exec(pod, skew_command, scenario["namespace"])
+                selected_container_name = get_container_name(pod, scenario["namespace"], container_name)
+                pod_exec(pod, skew_command, scenario["namespace"], selected_container_name)
+                if pod_exec_response is False:
```

Comment:
Think we missed capturing the pod_exec_response from the pod_exec command.

Reply:
Should be fixed now, PTAL